### PR TITLE
Genbank refactor tweak

### PR DESCRIPTION
**Pull request by florencio-mazzoldi**
- Project import now saves the project and returns the ID.
- New route for construct import. It returns a set of block ids for the roots, and the blocks structure.
- Blocks are now returned as an array, instead of an object
- Very initial genbank export functionality
- More tests

### CODE REVIEW

File: plugins/convert/genbank/index.js [hunk 1]
```diff
@@ -3,62 +3,76 @@ import * as fileSystem from '../../../server/utils/fileSystem';
...
+
+  //if you want to validate it afterwards (probably for debugging)
+  //This function will log errors about what fields failed validation
+  //you can pass in a second argument of true for this to throw if you want a stack trace
+  const isValid = BlockDefinition.validate(outputBlock);
+
+  //note that if you reject this promise, then Promise.all will reject as well. You probably dont want to do that.
+  //None of the blocks will be available on what is returned. Dont want you to run into that.
+  //Really, you probably just dont want this to happen ever...
+  if (!isValid) {
```
Comment (duncanmeech) [inline plugins/convert/genbank/index.js]: ?

File: plugins/convert/genbank/index.js [hunk 2]
```diff
@@ -106,7 +121,7 @@ const runCommand = (command, input, inputFile, outputFile) => {
       return new Promise((resolve, reject) => {
         exec(command, (err, stdout) => {
           if (err) {
-            console.log('Error executing command: '+err);
+            console.log('Error executing command: ' + err);
```
Comment (duncanmeech) [inline plugins/convert/genbank/index.js]: remove

**Comment by duncanmeech** [comment]
Looks good, :+1: when passing npm run lint

**Comment by florencio-mazzoldi** [comment]
@duncanmeech please review again.

**Pull request merged**